apps/meteor/client/views/admin/apps/AppMenu.js

```diff
@@ -68,7 +68,7 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 		(permissionsGranted) => {
 			setModal(null);
 
-			marketplaceActions[action]({ ...app, permissionsGranted }).then(() => {
+			marketplaceActions[action]({ ...app, permissionsGranted }).then(async () => {
```

Comment:
`async` is only needed when you're performing `await` inside a function. Since it is only calling a state and nothing async, then it is not needed.

Reply:
Or are you saying it shouldn't be here? (I read your message as you were asking it to have).

Reply:
> Or are you saying it shouldn't be here?

This is it. I just imagined the function was made async because, at some moment, it was necessary and the awaited call was removed later.